roles/marathon/templates/mesos-consul.json.j2

```diff
@@ -0,0 +1,21 @@
+{
+  "args": [
+    "--zk=zk://zookeeper.service.{{ consul_dns_domain }}:2181/mesos",
+    "--registry-ssl",
+    "--registry-ssl-verify=false",
```

Comment:
Now after granular security this needs to be conditional ... otherwise you can't disable ssl for consul